src/jdk.hotspot.agent/share/man/jhsdb.1

```diff
@@ -179,6 +179,16 @@ If the system property is not set, the default port 1099 is used.
 .RS
 .RE
 .TP
+.B \f[CB]\-\-disableregistry\f[R]
+Disable starting RMI registry on startup.
+This option overrides the system property
+\[aq]sun.jvm.hotspot.rmi.startRegistry\[aq].
+If not specified, RMI registry will be started on startup.
+Otherwise will it not start, the RMI registry (rmid, etc) is needed
+before starting debugd.
```

Comment:
Is this what you mean: `Otherwise it will not be started, and the already started RMI registry will be used instead.`